src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -161,7 +151,7 @@ t8_get_number_of_vtk_nodes (t8_element_shape_t eclass, int curved_flag)
  * TODO: Add Pyramids when they are merged into the dev branch.
  * */
 #if T8_WITH_VTK
-static void
```

Comment:
This function is independant of the linking with vtk. Therefore, we can delete the makros, I think.